tools/rdm/TestDefinitions.py

```diff
@@ -5071,7 +5071,7 @@ def Test(self):
       self.SetNotRun('Unable to determine pin code')
       return
 
-    self.AddIfSetSupported(self.NackSetResult(RDMNack.NR_FORMAT_ERROR))
+    self.AddIfSetSupported(self.NackSetResult(RDMNack.NR_DATA_OUT_OF_RANGE))
```

Comment:
Actually this was right before. Section 3.9 in E1.37-1 says "If a device receives a PIN code outside of this range, then it shall send a NACK with a NACK Reason Code of NR_FORMAT_ERROR." because NR_DATA_OUT_OF_RANGE is reserved for reporting the current PIN is incorrect.